src/main/java/com/flipkart/yak/config/PromptSchedule.java

```diff
@@ -0,0 +1,15 @@
+package com.flipkart.yak.config;
+
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+import lombok.experimental.SuperBuilder;
+import lombok.extern.jackson.Jacksonized;
+
+@Data
+@SuperBuilder
+@RequiredArgsConstructor
+@Jacksonized
+public class PromptSchedule {
```

Comment:
its not really a schedule. 
PromptCompactionLifespan you can say

Reply:
done